pystiche_papers/sanakoyeu_et_al_2018/_modules.py

```diff
@@ -184,3 +191,46 @@ def residual_block(channels: int) -> ResidualBlock:
     )
 
     return ResidualBlock(residual)
+
+
+def pairwise(iterable: Iterable) -> Iterator[Tuple[Any, Any]]:
+    a, b = itertools.tee(iterable)
+    next(b, None)
+    return zip(a, b)
+
+
+def discriminator_encoder_modules(
```

Comment:
Does this function serve any purpose, i.e. is there any use case where I would want to call that outside of the `Discriminator`? If yes, could you give me an example? If not, please move this into the constructor to `Discriminator`.